sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -4,14 +4,17 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
+  set(UNIFIED_RUNTIME_TAG e6343f4cca9a37b17bc63f3a81968ac3f486be8a)
```

Comment:
Why we want to `rollback` UR tag here?

* |   b38855e - (2023-10-05) Merge pull request #920 from jsji/localcopy - Piotr Balcer
|\ \
| * | 6a2c548 - (2023-10-02) [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path - Jinsong Ji (localcopy)
|/ /
* |   e6343f4 - (2023-10-04) Merge pull request #923 from pbalcer/l0-win-fix-build - Piotr Balcer